pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -7021,22 +7020,18 @@ int sched_process_exit_signal(struct bpf_raw_tracepoint_args *ctx)
 // That will allow to subscribe only to wanted events and make handing easier.
 statfunc bool should_trace_sock_set_state(int old_state, int new_state)
 {
+    // Listen.
     if (old_state == TCP_CLOSE && new_state == TCP_LISTEN) {
         return true;
     }
-    if (old_state == TCP_LISTEN && new_state == TCP_CLOSE) {
```

Comment:
What is the reason for no longer track socket close events? 🤔

Reply:
We are not sending tcp close events. It will double the amount of events, but these events right now are not actionable.